frontend/public/components/monitoring/dashboards/table.tsx

```diff
@@ -112,15 +112,26 @@ const Table: React.FC<Props> = ({ panel, pollInterval, queries }) => {
   if (error) {
     return <ErrorAlert message={error} />;
   }
-  // FIXME: It's a little unclear how to associate a column style with the data. This assumes the
```

Comment:
It turns out we need to look at `refId` on `panel.targets` to associate the column style with the data.